docs-src/cloud/migration.md

```diff
@@ -0,0 +1,88 @@
+---
+id: migration
+title: Migrating to Temporal Cloud
+sidebar_label: Migration
+description: Understand how to migrate to Temporal Cloud.
+tags:
+  - temporal cloud
+  - migration
+  - multi-cluster replication
+---
+
+Migrating to Temporal Cloud from a self-hosted Cluster will have different requirements depending on your usage.
+This guide provides some guidance based on our experience helping customers of all sizes successfully migrate.
+
+### What to expect from a migration
+
+Depending on your Workflows’ requirements, the migration process may be as simple as changing a few parameters, or may call for more extensive code changes.
+There are two aspects to consider when migrating: your Temporal Client connection code and your Workflow Executions.
+Here’s a high-level overview of what you can expect:
+
+- **Update the Client connection code in your Worker:** Your Client connection code requires several small changes to ensure Temporal Cloud becomes responsible for new Workflow Executions.
+- **Migrate Workflow Executions:** There are different approaches for new, running, and completed Workflow Executions.
+  - **New Workflow Executions:** After you’ve updated your Client connection code, Temporal Cloud will automatically be responsible for all new Workflow Executions.
+  - **Running Workflow Executions:** Short-running Workflows can often be drained and then started again on Temporal Cloud. Long-running Workflows that cannot be drained and might require you to implement more code changes to pass the state of the currently running Workflow to Temporal Cloud.
```

Comment:
I think the word "and" is unintended in "Long-running Workflows that cannot be drained **and** might require you to implement..."